fsm.go

```diff
@@ -273,11 +284,16 @@ func (s *stateMachine) isSuspect(id id) bool {
 	return ok
 }
 
-// isNews reports whether m contains new membership status information.
-func (s *stateMachine) isNews(m *message) bool {
+// isMemberNews reports whether m contains new membership status information.
+func (s *stateMachine) isMemberNews(m *message) bool {
 	if m == nil {
 		return false
 	}
+	switch m.Type {
+	case alive, suspected, failed:
```

Comment:
I'm a little unclear on the tradeoffs between this vs `if m.Type == userMsg`.

Reply:
Unwise and unnecessary overprotection. The simpler way is more readable.